packages/utilities/src/dom/elementContains.ts

```diff
@@ -0,0 +1,27 @@
+import { getParent } from './getParent';
```

Comment:
Splitting these functions into different files will cause bundle size increases for AMD I think--is that okay?